### ISSUE TITLE
Orders are stuck on "pending" status

### ISSUE THREAD

**Issue by JanMikes** [seberm/nette-paypal#23]
Hello, everything seem to be fine, but payments are stuck as "pending" and there are nothing to do with them.

Have you guys have same issue and possibly how have u solved this? I am not sure if this is related to the library or something is wrong with the paypal account.

**Comment by fmasa** [comment]
Are you calling `confirmExpressCheckout` method as suggested in [your comment](https://github.com/seberm/nette-paypal/issues/18#issuecomment-194242297) on different issue?

**Comment by JanMikes** [comment]
Yep. As well I can see the order in both customer's and seller's accounts, but it is has "Pending" status and money are not transfered. 

It seem to me that is is PayPal account settings related, not the library bug, so if you had any tips i would be grateful.

**Comment by JanMikes** [comment]
I added logging before calling `confirmExpressCheckout()`:

``` php
public function confirmOrder($control, $data)
{
    \Tracy\Debugger::log($data, "confirm-order");
    $control->confirmExpressCheckout();
}
```

I have noticed following data in all results, it might be the issue?

```
billingagreementacceptedstatus => "0"
checkoutStatus => "PaymentActionNotInitiated" (25)
```

**Comment by michalhlavka** [comment]
@Lexinek its because you are doing purchase in different currency than its set in your paypal merchant account

**Comment by JanMikes** [comment]
@michallhlavka that is actually not true

Anyways since this was big problem for me (receiving no money on merchant's account and having all payments as pending) and i was not able to find the reason causing it, i changed implementation to use metisfw/paypal library and everything is working for me now.

**Comment by michalhlavka** [comment]
@Lexinek ok, thanks for reaction, I will take a look at metisfw
